openedx/core/djangolib/model_mixins.py

```diff
@@ -12,3 +12,25 @@ def __init__(self, *args, **kwargs):  # pylint: disable=unused-argument
         Override to kill usage of this model.
         """
         raise TypeError("This model has been deprecated and should not be used.")
+
+
+class DeletableByUserValue(object):
```

Comment:
The class name should contain `mixin` word in it to be consistent

Reply:
:) Didn't we have a conversation yesterday about "Deletable" being sufficient for signifying that this is a Mixin?